Paskab012/cartix-web1
# Disable enable

#### Hello Reviewer ✋🏽 
## Description of changes made ♻️
Implement toggle button

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/Paskab012/cartix-web1.git)
- Check out this branch by `disable_enable`
- Run `yarn or npm install to install the project dependencies
- Run `npm run start:cartix` to start the development server

#### Checklist


#### Screenshots

|        BEFORE                  |                     AFTER       |
| :----------------------------: | :-----------------------------: |
|                                | Note: You can even provide a    |
|                                | screenshot                      |
